packages/protoplugin/src/run-node.ts

```diff
@@ -75,18 +78,3 @@ function readBytes(stream: ReadStream): Promise<Uint8Array> {
     });
   });
 }
-
-/**
- * Node.js buffers stdout, and process.exit() will truncate output.
- * As a workaround, we set the stream to blocking via a private API.
- * See https://github.com/timostamm/protobuf-ts/issues/134
- * See https://github.com/nodejs/node/issues/6456
- */
-function setBlockingStdout(): void {
```

Comment:
Tested both with and without this workaround and it works without it so we can probably give this a shot to see if we can simplify the code.